test/groovy/CloudFoundryDeployTest.groovy

```diff
@@ -104,17 +105,17 @@ class CloudFoundryDeployTest extends BasePiperTest {
                 cloudFoundryDeploy: []
             ]
         ]
-
+        nullScript.commonPipelineEnvironment.setBuildTool('mta')
         stepRule.step.cloudFoundryDeploy([
             script: nullScript,
             juStabUtils: utils,
             jenkinsUtilsStub: new JenkinsUtilsMock(),
             deployTool: '',
+            mtaPath: 'target/test.mtar',
```

Comment:
Why do we need to change this?

Reply:
The test case tests other behavior than before, thus a valid configuration is required.